image/scanner/rhel/konflux.Dockerfile

```diff
@@ -75,3 +68,33 @@ RUN microdnf upgrade --nobest && \
 USER 65534:65534
 
 ENTRYPOINT ["/entrypoint.sh"]
+
+# Scanner Slim
+FROM scanner-common as scanner-slim
+
+LABEL \
+    com.redhat.component="rhacs-scanner-slim-container" \
+    io.k8s.display-name="scanner-slim" \
+    name="rhacs-scanner-slim-rhel8"
+
+ENV ROX_SLIM_MODE="true"
+
+# Extension of common to full builder
+FROM builder-common as builder
+
+RUN unzip -j blob-k8s-definitions.zip -d image/scanner/dump/k8s_definitions && \
+    unzip -j blob-nvd-definitions.zip -d image/scanner/dump/nvd_definitions
```

Comment:
Well, you can still have these in `builder-common`. Just don't copy to `scanner-common` (you already changed it not to copy there). This `builder` stage becomes redundant and `builder-common` can be renamed back to `builder`. WDYT?

Reply:
Moved to a common stage and added both files to the prefetching step in scanner-slim. It's clearer to have the differentiation just in the Dockerfile, not in many places.

Reply:
* https://github.com/stackrox/scanner/pull/1429/commits/87d959b340a75cfb83f4d65c0f8d96fd5594d61a
* https://github.com/stackrox/scanner/pull/1429/commits/d00818d21797414f8193e0e0880c5ec2795ecbc0